client/fingerprint/vault.go

```diff
@@ -17,20 +17,15 @@ import (
 	vapi "github.com/hashicorp/vault/api"
 )
 
-const (
-	vaultAvailable   = "available"
-	vaultUnavailable = "unavailable"
-)
```

Comment:
Note to reviewers: this was dead code after https://github.com/hashicorp/nomad/pull/18253 that I missed